istqb-ct-pt-complete-guide.md

```
@@ -0,0 +1,1415 @@
+## 🌟 Chapter 0: CT-PT 概要と資格ロードマップ {#chapter-0}
+
+### 0.1 この資格とは？
+
+flowchart TD
+    subgraph Advanced_Expert [Advanced / Expert Level]
+        TTA[CTAL-TTA]
+        TM[CTAL-TM]
+        TAE[CTAL-TAE]
+    end
+
+    subgraph Specialist [Specialist Stream]
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)